exercises/concept/sorting-room/.meta/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "blurb": "Learn about type casting and type assertions in the sorting room.",
+  "authors": [
+    "jmrunkle"
+  ],
```

Comment:
Maybe I ("junedev") could get a contributor slot here for helping with exercise structure/adding the fancy number part?

Reply:
Oh for sure, let me do that super quick. Thanks!